### DISCUSSION TITLE
Q: Can anyone please point me to their dotfiles, so I can see some example configuration?

### DISCUSSION THREAD

**Discussion by hristod** [lopi-py/luau-lsp.nvim#33] [Q&A] [answered]
Maybe it's me, but the dev experience is much worse than VSCode, and I believe this is due to misconfiguration on my end. Please, if anyone can point me to their .dotfiles, that would be very helpful! Thank you!

**Comment by lopi-py** [comment]
Can you please elaborate about the experience being worse than vscode?

A time ago I created [kickstart-roblox.nvim](https://github.com/lopi-py/kickstart-roblox.nvim) to guide newcomers to roblox dev with neovim. Here is my [neovim config](https://github.com/lopi-py/nvim-config)

**Reply by hristod** [reply]
Hanks a lo for converting he issue o discussion. I will know better next time :)
I will take a look at the kickstart repo, and I will compare my experience with it. For now, I ironed out my config until only having these two issues:

- When I import a service like so: 
`local BallService = require(ReplicatedStorage:WaitForChild("Shared"):WaitForChild("BallService"))`, I get the following error:

```
'BallService' from '.../code/roblox/ball_duel/src/shared/BallService/init.luau [game/ReplicatedStorage/Shared/BallService]'

could not be converted into
'BallService' from '.../code/roblox/ball_duel/src/shared/BallService/init.luau [game/ReplicatedStorage/Shared/BallService]'
caused by:
` Property 'IsExcept' is not compatible.
```

- Roblox globals are not recognised. I have many errors saying thins like "TypeError: Unknown global 'task" and and such.
- Also, many times i poress `shift+k`, sometimes I get an `*error-type*` instead of the info. If I do this a few times, at one pint it works :)

**Comment by hristod** [comment]
BTW, I think it would be useful if you add a link to [kickstart-roblox.nvim](https://github.com/lopi-py/kickstart-roblox.nvim) to the readme. Just an idea :) I can do a PR if you want.

**Reply by lopi-py** [reply]
> BTW, I think it would be useful if you add a link to [kickstart-roblox.nvim](https://github.com/lopi-py/kickstart-roblox.nvim) to the readme. Just an idea :) I can do a PR if you want.

Eventually, I will add a wiki page for stuff like this 😅

**Comment by hristod** [comment, marked as answer]
An update: I figured out how to fix my issue. It was caused by `luau-lsp` being loaded twice: once by the plugin, and once by `mason-lspconfig`. By removing the second attachment, it all works great now. Thank you for the help, @lopi-py !
